identity/client/src/pages/mappings/List.tsx

```diff
@@ -68,9 +64,10 @@ const List: FC = () => {
       <EntityList
         data={mappingSearchResults == null ? [] : mappingSearchResults.items}
         headers={[
-          { header: t("Mapping Key"), key: "mappingKey" },
-          { header: t("Claim Name"), key: "claimName" },
-          { header: t("Claim Value"), key: "claimValue" },
+          { header: t("Mapping ID"), key: "mappingKey" },
+          { header: t("Mapping name"), key: "name" },
+          { header: t("Claim name"), key: "claimName" },
+          { header: t("Claim value"), key: "claimValue" },
```

Comment:
These should probably have translation keys. You could also create a new translation file for mappings.